tests/test_project_management.py

```diff
@@ -1065,6 +1082,12 @@ def test_delete_sha_certificates_not_found(self, android_app):
         assert excinfo.value.detail is not None
         assert len(recorder) == 1
 
+    def test_raises_if_app_has_no_project_id(self):
```

Comment:
Can we use `testutils.run_without_project_id()` here as well?

Reply:
Done.